bundle/edu.gemini.sp.vcs/src/test/scala/edu/gemini/sp/vcs/diff/TestEnv.scala

```diff
@@ -23,18 +25,34 @@ case class TestPeer(odb: IDBDatabaseService, server: VcsServer, service: Princip
   def vcs(p: Principal): Vcs =
     new Vcs(VcsAction(Set(p)), server, _ => service(p))
 
-  val staffVcs: Vcs = vcs(StaffPrincipal.Gemini)
+  val superStaffVcs: Vcs = vcs(StaffPrincipal.Gemini)
 
   def shutdown(): Unit = odb.getDBAdmin.shutdown()
 
+  def spDataObject: SPProgram = prog.getDataObject.asInstanceOf[SPProgram]
+
   // Lookup the test program
   def prog: ISPProgram = odb.lookupProgram(TestEnv.Key)
 
+  def set(update: SPProgram => Unit): Unit =
+    spDataObject <| update |> prog.setDataObject
```

Comment:
Some concepts just don't map to a word or two very well.  For example I think `<|` is actually clearer than `unsafeTap` once you are initiated.  According to the doc:

> Mostly for use with dodgy libraries that give you values that need additional initialization or mutation before they're valid to use.

I think the science program model qualifies.